src/components/CategoryHeader.jsx

```diff
@@ -1,7 +1,11 @@
 import React from "react";
 import { Menu, Segment } from "semantic-ui-react";
 import { connect } from "react-redux";
-import { SELECT_CATEGORY } from "../state/actions/actionTypes";
+import {
+  SELECT_CATEGORY,
+  LOGOUT,
+  SHOW_LOGIN_FORM
+} from "../state/actions/actionTypes";
```

Comment:
I would probably use to import everything from the actiontypes when it is so many, 
```import as * from '../state/actions/actionTypes' ```